services/credential-server-ui/src/pages/RevocationPage.tsx

```diff
@@ -48,11 +51,18 @@ const RevocationPage: React.FC = () => {
   };
 
   const handleRevokeCredential = async (values: any) => {
-    await axios.post(`${config.endpoint}${config.path.revokeCredential}`, {
-      credentialId: values.selectedCredential,
-      holder: values.selectedContact
-    });
-    await handleGetContactCredentials(values.selectedContact);
+    try {
```

Comment:
again dont think we need the try catch as we are't doing anything different except console logging and the browser env won't freeze up if this throws